handy-win32src-0.95-patched/core/eeprom.h

```diff
@@ -11,7 +11,8 @@ enum {EE_NONE=0, EE_START, EE_DATA, EE_BUSY, EE_WAIT};
 class CEEPROM : public CLynxBase
 {
 
-   // Function members
+  
+#define min(x,y) x>y?y:x
```

Comment:
I would add parenthesis: #define min(x,y) (x>y?y:x)

Reply:
edit: what about just replacing min with __min, that's the one from windows stdlib apparently.

Reply:
I would even go for:
#define min(x,y) ((x)>(y)?y:x)
And: No Compiler/platform dependency if not needed.